### PR TITLE
ENH: Added warning

### PR DESCRIPTION
**Pull request by stuwilkins**
Attn @mpmdean 

Warning added, as per #45

### CODE REVIEW

**Comment by stuwilkins** [comment]
Attn @mpmdean added function to normalize back to what is expected if the stack had no

File: csxtools/image/stack.py [hunk 1]
```diff
@@ -32,17 +35,35 @@ def stacksum(array):
...
     Returns
     -------
     tuple
         tuple of 2 arrays of the sum and number of points in the sum
     """
     X, Y = extimage.stackprocess(array, 0)
+
+    total_elements = array.size / (array.shape[-1] * array.shape[-2])
+
+    if np.sum(Y != total_elements):
```
Comment (tacaswell) [inline csxtools/image/stack.py]: should this be `if not norm and np.sum(Y != total_elements):` ?

Comment (stuwilkins) [inline csxtools/image/stack.py]: yes... fixing

**Comment by stuwilkins** [comment]
@mpmdean I moved this so the stack automatically normalizes for missing data points. Docs are updated.

**Pull request merged**